tasks/libs/ciproviders/github_api.py

```diff
@@ -491,9 +493,11 @@ def create_release_pr(title, base_branch, target_branch, version, changelog_pr=F
         "qa/no-code-change",
         "team/agent-delivery",
         "team/agent-release-management",
-        "category/release_operations",
     ]
 
+    if not laststable_pr:
+        labels.append("category/release_operations")
```

Comment:
I think that we actually don't need this label any more. It used to be required in the past but we stopped using it recently. So the logic behind the `laststable_pr` will not be needed.